docs/node/testnet/upgrade-log.md

```diff
@@ -57,7 +57,12 @@ upgrade epoch.
 5. Ensure your node will remain stopped by disabling auto-starting via your
    process manager (e.g., [systemd] or [Supervisor])
 
-6. [Wipe state]. This must be performed _before_ replacing the Oasis Node
+6. Back up the entire directory of your node. Verify that the backup includes
+the following folders:
+- for consensus: `tendermint/abci-state` and `tendermint/data`
+- for runtimes: `runtimes/*/mkvs_storage.badger.db`
```

Comment:
Actually for confidential runtimes the `worker-local-storage.badger.db` is also critically important.

Reply:
Added